+exp/start_amt.m

```diff
@@ -41,15 +41,10 @@
 %刺激数
 objectNum=12;
 
-%注视点和context设置
-crossWidth = 2; 
-crossColor = [0 0 0];
-crossLengthPixels = 20;
-crossLines = [-crossLengthPixels crossLengthPixels 0 0; 0 0 -crossLengthPixels crossLengthPixels];%注视点设置
-rectlineColor = [0 0 0];
+%context设置
 contextTexture = cell(1,4);
 for context = 1:4
-    fileName = [pwd '/stimuli/context/context' num2str(context) '.jpg'];
+    fileName = [pwd '/stimuli/amt/context/context' num2str(context) '.jpg'];
```

Comment:
This should be:

`fileName = fullfile('stimuli', 'amt', 'context', sprintf('context%d.jpg', context));`